checks/class-title-check.php

```diff
@@ -50,7 +50,7 @@ public function check( $php_files, $css_files, $other_files ) {
 			}
 
 			// Look for anything that looks like <svg>...</svg> and exclude it (inline svg's have titles too).
-			$file_content = preg_replace( '/<svg.*>.*<\/svg>/s', '', $file_content );
+			$file_content = preg_replace( '/<svg.*>.*<\/svg>/i', '', $file_content );
```

Comment:
I think we should use the 2 flags `i` and `s`.

```suggestion
			$file_content = preg_replace( '/<svg.*>.*<\/svg>/is', '', $file_content );
```

Reply:
Hi @matiasbenedetto,
Upon further reflection, I agree with your suggestion regarding the need for the 's' modifier. However, I discovered that using 's' on its own does not work since it nullifies the result of the modified line. 
After a second round of troubleshooting, including refactoring the regex and the complete logic, I found that combining 's' with the 'U' modifier seems to be the most universal and effective solution. 
I have updated my commit accordingly to reflect this change.